keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/CheckScheduler.kt

```diff
@@ -43,25 +47,49 @@ class ResourceCheckScheduler(
   @Scheduled(fixedDelayString = "\${keel.resource-check.frequency:PT1S}")
   fun checkResources() {
     if (enabled) {
-      log.debug("Starting scheduled validation…")
-      publisher.publishEvent(ScheduledCheckStarting)
+      log.debug("Starting scheduled resource validation…")
+      publisher.publishEvent(ScheduledResourceCheckStarting)
```

Comment:
Changed event name because there are scheduled checks for the environment now too